typescript/winston-cloudwatch.d.ts

```diff
@@ -1,44 +1,37 @@
-import * as winston from 'winston';
+import * as winston from "winston";
+import * as Transport from "winston-transport";
 
-import { CloudWatch, CloudWatchLogs } from 'aws-sdk';
+import { CloudWatch, CloudWatchLogs } from "aws-sdk";
 
-interface CloudWatchIntegration {
```

Comment:
You've removed lots of functions from here, I am not saying it's wrong, I'd like to have a code example on how you use this with TypeScript as I've never used this library with it.